### PR TITLE
Default linkable for last crumb should be false by default

### LINKED ISSUE #21: [Feature Request]: Linkable false issue in child views
If the route is "foo.baz" and setting "baz" as "linkable:false", and having "edit" route as child of "baz" renders the breadcrumb as "foo > baz > edit" where
foo - linkable
baz - non linkable
edit - linkable.

### PR DESCRIPTION
**Pull request by abhilashlr**
This PR ensures that the default linkable for the last crumb will be always false unless specified from the route. Will close #21

### CODE REVIEW

File: addon/components/bread-crumbs.js [hunk 1]
```diff
@@ -78,12 +78,16 @@ export default Component.extend({
...
   _lookupBreadCrumb(routeNames, filteredRouteNames) {
-    const defaultLinkable = get(this, 'linkable');
+    let defaultLinkable = get(this, 'linkable');
+    const pathLength = routeNames.length;
     const breadCrumbs = filteredRouteNames.map((name, index) => {
       const path = this._guessRoutePath(routeNames, name, index);
       let breadCrumb = this._lookupRoute(path).getWithDefault('breadCrumb', undefined);
       const breadCrumbType = typeOf(breadCrumb);
 
+      if(index === pathLength - 1) {
```
Comment (poteto) [inline addon/components/bread-crumbs.js]: Needs a space between `if ` and `(`

File: addon/components/bread-crumbs.js [hunk 2]
```diff
@@ -78,12 +78,16 @@ export default Component.extend({
...
+        defaultLinkable = false;
+      }
```
Comment (poteto) [inline addon/components/bread-crumbs.js]: New line please

File: addon/components/bread-crumbs.js (hunk 1 above)
Comment (abhilashlr) [inline addon/components/bread-crumbs.js]: Added the space.

File: addon/components/bread-crumbs.js (hunk 2 above)
Comment (abhilashlr) [inline addon/components/bread-crumbs.js]: I guess the CI failed due to space. Now it is cleared. But still the build fails in another test.

**Comment by abhilashlr** [comment]
@poteto any plans on merging this PR?

**Comment by poteto** [comment]
Yes, when the tests pass. Because the last link in the breadcrumb is no longer a link, it is breaking another test ("bread-crumbs component updates when dynamic segments change"), so you will need to update the selector in use there (it was expecting an `a` tag)

**Comment by abhilashlr** [comment]
Oki. let me check the test case

**Comment by abhilashlr** [comment]
@poteto the test cases are updated.! sorry about missing them earlier

**Comment by poteto** [comment]
No problem! If you could squash this down into 1 commit, I'll merge it in :) Thanks for contributing!

**Pull request closed without merging**